backend/src/modules/auth/authRoutes.js

```diff
@@ -165,6 +169,32 @@ router.put(
       return res.status(422).json({ error: '422: Not existing user' });
     }
 
+    const userDetail = await dbConnection.query(
+      'SELECT email FROM users WHERE user_id = ? AND active = true;',
+      [userId]
+    );
+
+    if (userDetail.length > 0){
+      const { email } = userDetail[0];
+      const lobbyInvitation = await dbConnection.query(
+        'SELECT invitation_id, lobby_id FROM invitation WHERE email = ? AND approved = true;',
+        [email]
+      );
+      if (lobbyInvitation && lobbyInvitation.length > 0) {
+        lobbyInvitation.forEach(async invitation => {
```

Comment:
Pouzivej map misto forEach.. Vyjde to na stejno, ale map je beznejsi obecne v pouzivani.